Remora.Commands/Trees/CommandTreeBuilder.cs

```diff
@@ -63,6 +86,15 @@ public void RegisterModule(Type commandModule)
         }
     }
 
+    /// <summary>
+    /// Registers a command with the builder.
+    /// </summary>
+    /// <param name="builder">The builder to register.</param>
+    public void RegisterBuilder(OneOf<CommandBuilder, GroupBuilder> builder)
```

Comment:
This name is awkward

Reply:
It is a bit cumbersome to deal with. Open to suggestions!

Reply:
RegisterCommandBuilder